transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -360,23 +360,13 @@ public String toString() {
         SocketAddress remoteAddr = remoteAddress();
         SocketAddress localAddr = localAddress();
         if (remoteAddr != null) {
-            SocketAddress srcAddr;
-            SocketAddress dstAddr;
-            if (parent == null) {
-                srcAddr = localAddr;
-                dstAddr = remoteAddr;
-            } else {
-                srcAddr = remoteAddr;
-                dstAddr = localAddr;
-            }
-
             StringBuilder buf = new StringBuilder(96)
                 .append("[id: 0x")
                 .append(id.asShortText())
                 .append(", ")
-                .append(srcAddr)
-                .append(active? " => " : " :> ")
-                .append(dstAddr)
+                .append(localAddr)
+                .append(active? " - " : " ! ")
```

Comment:
@trustin  sorry, I misunderstand them: "Left" and "Right" instead of "Local" and "Remote".
so which is best?
[id: 0x9f79a8ce, L /172.16.2.216:45678 - R /172.16.1.10:49839].
[id: 0x9f79a8ce, L< /172.16.2.216:45678> - R</172.16.1.10:49839>].
[id: 0x9f79a8ce, Local< /172.16.2.216:45678> - Remote</172.16.1.10:49839>].